region/maridia/outer/Main Street.json

```diff
@@ -3920,6 +4137,79 @@
       },
       "devNote": "This depends on how far the Morph tunnel is expanded, but this is both the minimum distance and the distance Map Rando has expanded."
     },
+    {
+      "link": [5, 5],
+      "name": "Sciser Farm",
+      "requires": [
+        "Morph",
+        {"resetRoom": {"nodes": [5]}},
+        {"or": [
+          {"and": [
+            "Gravity",
+            {"or": [
+              {"and": [
+                "ScrewAttack",
+                {"cycleFrames": 480}
+              ]},
+              {"and": [
+                "canPseudoScrew",
+                {"cycleFrames": 480}
+              ]},
+              {"and": [
+                "Plasma",
+                {"cycleFrames": 480}
+              ]},
+              {"and": [
+                "Wave",
+                {"cycleFrames": 480}
+              ]},
+              {"and": [
+                "Spazer",
+                {"cycleFrames": 480}
+              ]},
+              {"and": [
+                "Charge",
+                "canDodgeWhileShooting",
+                {"cycleFrames": 480}
+              ]},
+              {"and": [
+                "Ice",
+                {"cycleFrames": 520}
+              ]},
+              {"and": [
+                "canFarmWhileShooting",
+                {"cycleFrames": 540}
+              ]},
+              {"cycleFrames": 1620}    
+            ]}
+          ]},
+          {"and": [
+            "Plasma",
+            {"cycleFrames": 690}
+          ]},
+          {"and": [
+            "Wave",
+            {"cycleFrames": 690}
+          ]},
+          {"and": [
+            "Spazer",
+            {"cycleFrames": 690}
+          ]},
+          {"and": [
+            "Ice",
+            {"cycleFrames": 710}
+          ]},
+          {"and": [
+            "Charge",
```

Comment:
I get sub 12 without charge.  Doing two shots per jump makes it easier than with gravity.
With charge I can kill the crab too fast and have to wait for the drop to move itself closer so there is no time gain.

With gravity, charge saved me 30f